gramex/ml.py

```diff
@@ -377,5 +379,56 @@ def translater(handler, source='en', target='nl', key=None, cache=None, api='goo
     raise Return(result.to_json(orient='records'))
 
 
+_languagetool = {
+    'defaults': {k: v for k, v in variables.items() if k.startswith('LT_')},
+    'installed': os.path.isdir(variables['LT_CWD'])
+}
+
+
+@coroutine
+def languagetool(handler, *args, **kwargs):
+    merge(kwargs, _languagetool['defaults'], mode='setdefault')
+    # If languagetool is not installed, install it
+    if not _languagetool['installed']:
+        if not os.path.isdir(kwargs['LT_TARGET']):
+            os.makedirs(kwargs['LT_TARGET'])
+        import requests, zipfile, io        # noqa
+        src = kwargs['LT_SRC'].format(**kwargs)
+        app_log.info('Downloading languagetools from %s', src)
+        stream = io.BytesIO(requests.get(src).content)
+        app_log.info('Unzipping languagetools to %s', kwargs['LT_TARGET'])
+        zipfile.ZipFile(stream).extractall(kwargs['LT_TARGET'])
+        _languagetool['installed'] = True
+
+    # Process request via languagetool
+    client = AsyncHTTPClient()
+    tries = int(handler.get_argument('tries', 3))
```

Comment:
For now, please hard-code this to 1. When required, we'll generalize. I'd like to see what kind of failures we get, and why.